en/flight_controller/cuav_v5_nano.md

```diff
@@ -84,8 +84,8 @@ make px4_fmu-v5_default
 
 ## Voltage Ratings
 
-*V5 NANO* can be Two-fold redundancy on the power supply if two power sources are supplied.
-The three power rails are: **POWER1**and **USB**.
+*V5 NANO* can be two-fold redundancy on the power supply if two power sources are supplied.
```

Comment:
@davids5 FYI, I changed this to say two power rails, since we only mention two, and we don't support PM2 on PX4.

Reply:
I think of redundancy as a aspect of flight. Do we fly will USB? No. So this is not accurate. The Nano has only one PM that will power the FMU period. Have a look at the schematics.

Reply:
This is not accurate. - see other comments.
The Nano can be powered from USB and PM1 but not PM2

Reply:
Thanks. Could not agree more.  Was trying to get CUAV to clarify this. Fixed now. Note that the terminology I try to use for the ports is what is screenprinted on the case. So I will refer to this as `Power`. Fixed.